docs/self_hosted/optional_features.md

```diff
@@ -0,0 +1,23 @@
+---
+id: optional_features
+title: Optional features
+sidebar_label: Optional features
+---
+
+Mia-Platform Console offers some optional features which are disabled by default and can be enable as needed.
+
+### Google Analytics Tracking
+
+Google Analytics Tracking can be enabled activating the feature toggle `ENABLE_GOOGLE_TAG_MANAGER`. It can be activated by a Company Owner using the CMS, in the "Feature Activations" section.
```

Comment:
not by company owners but by Console Super Users; id make a generic paragraph above explaning only once how to activate any feature with the CMS and here only provide the information to be put in the CMS